roqoqo/tests/integration/operations/mod.rs

```diff
@@ -30,6 +30,9 @@ mod supported_version;
 
 mod bosonic_operations;
 
+mod spin_boson_operations;
+pub use spin_boson_operations::*;
```

Comment:
This line can be removed

Reply:
Also I believe this should have the feature gate too

Reply:
Fixed